### PR TITLE
refactor: removed moveto command

### LINKED ISSUE #6809: POST /session/<sessionid>/moveto did not match a known command
## To Reproduce

Detailed steps to reproduce the behavior:

1.  Clone https://github.com/cambiph/se-debug.git
2. Perform `npm install`
3. Start the docker containers using `docker-compose up`
4. Run `grunt builddocker`

## Expected behavior

Firefox browser is opened and `www.angular.io`is loaded.
Then, the mouse is moved to the element with classname `card-text-container`

## Test script or set of commands reproducing this issue

Please provide a test script to reproduce the issue you are reporting, if the 
setup is more complex, GitHub repo links with are also OK.

Issues without a reproduction script are likely to stall and eventually be closed.

See above

## Environment

Latest build from https://hub.docker.com/u/selenium/
Protractor version used is the latest beta which includes the latest selenium JS bindings (4.0.0-alpha1)
More details can be found in the Twitter discussion: https://twitter.com/shs96c/status/1078794260265218048

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Fixes #6809

### Motivation and Context

We will be removing this legacy `moveto` action as w3c had implemented actions API which can be used to move the mouse.

- Removed [mouseMove](https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/node/selenium-webdriver/lib/actions.js#L143) function, as this method consumes command `LEGACY_ACTION_MOUSE_MOVE`.
- Also removed `Action.Type.POINTER_MOVE`, as that switch case was using `LEGACY_ACTION_MOUSE_MOVE`, and we are deprecating it. As we had removed `Action.Type.POINTER_MOVE`, so now we don't need [POINTER_MOVE](https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/node/selenium-webdriver/lib/input.js#L207) as well.
- Removed [scheduleMouseAction_](https://github.com/SeleniumHQ/selenium/blob/d61c780f728dc82894a4b576007a9fd30ba542c1/javascript/node/selenium-webdriver/lib/actions.js#L180) function, as this method was using `mouseMove`.
- Removed [click](https://github.com/SeleniumHQ/selenium/blob/d61c780f728dc82894a4b576007a9fd30ba542c1/javascript/node/selenium-webdriver/lib/actions.js#L297), [doubleClick](https://github.com/SeleniumHQ/selenium/blob/d61c780f728dc82894a4b576007a9fd30ba542c1/javascript/node/selenium-webdriver/lib/actions.js#L326), [mouseDown](https://github.com/SeleniumHQ/selenium/blob/d61c780f728dc82894a4b576007a9fd30ba542c1/javascript/node/selenium-webdriver/lib/actions.js#L226), [mouseUp](https://github.com/SeleniumHQ/selenium/blob/d61c780f728dc82894a4b576007a9fd30ba542c1/javascript/node/selenium-webdriver/lib/actions.js#L256), [dragAndDrop](https://github.com/SeleniumHQ/selenium/blob/d61c780f728dc82894a4b576007a9fd30ba542c1/javascript/node/selenium-webdriver/lib/actions.js#L276) and [move](https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/node/selenium-webdriver/lib/input.js#L836) functions as these functions were using `scheduleMouseAction_`

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by AutomatedTester** [comment]
@vaibhavsingh97 That method can be removed too and then clean up the tests.

**Comment by vaibhavsingh97** [comment]
@AutomatedTester thanks for the help 🙇 
Can you please review it, I would love to address your comments. I had gone ahead and updated the PR description explaining the changes I had made.

File: javascript/node/selenium-webdriver/lib/actions.js [hunk 1]
```diff
@@ -125,213 +125,6 @@ class LegacyActionSequence {
...
-    return this
-
-    /** @param {{x: number, y: number}} offset The offset to use. */
-    function setOffset(offset) {
-      cmd.setParameter('xoffset', offset.x || 0)
-      cmd.setParameter('yoffset', offset.y || 0)
-    }
-  }
-
-  /**
```
Comment (AutomatedTester) [inline javascript/node/selenium-webdriver/lib/actions.js:165]: For this PR we were only looking at `mouseMove`. The other files may need tweaks where they call `mouseMove`

File: javascript/node/selenium-webdriver/test/actions_test.js [hunk 2]
```diff
@@ -50,115 +50,6 @@ test.suite(function (env) {
       return driver.quit()
     })
 
-    it('click(element)', async function () {
```
Comment (AutomatedTester) [inline javascript/node/selenium-webdriver/test/actions_test.js:53]: I am not sure that this test are related to the change

File: javascript/node/selenium-webdriver/test/actions_test.js [hunk 3]
```diff
@@ -50,115 +50,6 @@ test.suite(function (env) {
...
-      await driver.get(fileServer.whereIs('/data/actions/click.html'))
-
-      let box = await driver.findElement(By.id('box'))
-      assert.equal(await box.getAttribute('class'), '')
-
-      await driver.actions().click(box).perform()
-      assert.equal(await box.getAttribute('class'), 'green')
-    })
-
-    it('click(element) clicks in center of element', async function () {
```
Comment (AutomatedTester) [inline javascript/node/selenium-webdriver/test/actions_test.js:63]: I am not sure that this test are related to the change

File: javascript/node/selenium-webdriver/test/actions_test.js [hunk 4]
```diff
@@ -204,16 +95,5 @@ test.suite(function (env) {
 
       assert.equal(await el.getAttribute('value'), 'foOBar')
     })
-
-    it('can interact with simple form elements', async function () {
```
Comment (AutomatedTester) [inline javascript/node/selenium-webdriver/test/actions_test.js:208]: I am not sure that this test are related to the change

File: javascript/node/selenium-webdriver/test/lib/webdriver_test.js [hunk 5]
```diff
@@ -1526,76 +1526,6 @@ describe('WebDriver', function () {
       .then(fail, (v) => assert.strictEqual(v, e))
   })
 
-  describe('actions()', function () {
```
Comment (AutomatedTester) [inline javascript/node/selenium-webdriver/test/lib/webdriver_test.js:1529]: These are not related to the legacy move

**Comment by harsha509** [comment]
Hi @vaibhavsingh97 ,

Did you get a chance to look into the above comments ?

Thanks,
Harsha

**Comment by vaibhavsingh97** [comment]
> Hi @vaibhavsingh97 ,
> 
> Did you get a chance to look into the above comments ?
> 
> Thanks,
> Harsha

Apologies I couldn't get back on time for this :sweat_smile:. I have some issues with JS tests, will be fixing those first. I will try to quickly resolve this. 

Thanks much for understanding 🙏 🙇

**Comment by harsha509** [comment]
Hi @vaibhavsingh97 ,

Thanks for the PR.

As I didn't got the reply soon, so I replaced this PR is replaced with PR https://github.com/SeleniumHQ/selenium/pull/9362 as removing legacy actions will also remove `bridge` actions and removing respective tests.

**Comment by diemol** [comment]
Closing as the changes were merged in a different PR.
